inbc-program/inbc/parser.py

```diff
@@ -204,12 +211,13 @@ def parse_query_args(self) -> None:
         parser_query.set_defaults(func=query)
 
 
-def _get_password(args) -> Optional[str]:
-    password = None
-    if args.username:
-        password = getpass.getpass("Please provide the password: ")
-    return password
-
+def _get_password(username, dockerusername) -> Optional[str]:
+        password = None
+        if username:
+           password = getpass.getpass("Please provide the password: ")
+        if dockerusername:
+           password = getpass.getpass("Please provide the docker password: ")
+        return password
```

Comment:
what if both are provided?  the docker one will overwrite the previous.  Probably need to check that only one is provided, or make a separate method for docker vs user.

Reply:
@nmgaston no it won't rewrite because we call the function with only one parameter each time, like here

'password': _get_password(args.username, None),
'dockerPassword': _get_password(None, args.dockerusername)

So it wouldn't rewrite

Reply:
@nmgaston changed